lib/chef/knife/bootstrap_windows_base.rb

```diff
@@ -319,6 +324,26 @@ def bootstrap(proto=nil)
         end
 
         wait_for_remote_response( config[:auth_timeout].to_i )
+
```

Comment:
While the bootstrap method is already too long. It would be a good ides to extract this bit to a separate method.